source/tutorials/write-delete-documents.txt

```diff
@@ -78,26 +77,29 @@ with a ``name`` value of ``"Ready Penny Inn"``:
 .. code-block:: java
    :copyable: true
 
-   Mono.from(restaurants.deleteOne(eq("name", "Ready Penny Inn"))).block();
+   Publisher<DeleteResult> deletePublisher = restaurants.deleteOne(
+           eq("name", "Ready Penny Inn"));
+   Mono.from(deletePublisher).block();
 
 Delete Multiple Documents
 -------------------------
 
-To delete multiple documents from a MongoDB collection, use the static
-``Mono.from`` method from the ``Mono`` ``Publisher`` to call the ``deleteMany()``
-method and pass in your query filter. The ``Mono`` ``Publisher`` is a
+To delete multiple documents from a MongoDB collection, call the ``deleteMany()``
+method and pass in your query filter. Then, pass the ``deleteMany()`` result to the static
+``Mono.from`` method from the ``Mono Publisher``. The ``Mono Publisher`` is a
 ``Publisher`` implementation from the Project Reactor library. To learn more
-about ``Mono``, see `Mono
-<https://projectreactor.io/docs/core/release/reference/#mono>`__ in the Project
+about the ``Mono`` class, see `Mono <https://projectreactor.io/docs/core/release/reference/#mono>`__ in the Project
 Reactor documentation.
 
-The following example removes all documents in the ``restaurants`` collection with a ``borough`` value of ``"Brooklyn"``:
+The following example removes all documents in the ``restaurants`` collection
+with a ``borough`` value of ``"Brooklyn"``:
```

Comment:
S: to [avoid with](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using):
```suggestion
that have a ``borough`` value of ``"Brooklyn"``:
```